lib/hydra/works/services/generic_file/persist_derivative.rb

```diff
@@ -9,13 +9,13 @@ class PersistDerivative < Hydra::Derivatives::PersistOutputFileService
     # The purpose of this Service is for use as an alternative to the default Hydra::Derivatives::PersistOutputFileService.  It's necessary because the default behavior in Hydra::Derivatives assumes that you're using LDP Basic Containment. Hydra::Works::GenericFiles use IndirectContainment.  This Service handles that case.
     # 
     # @param [Hydra::Works::GenericFile::Base] object the file will be added to
-    # @param [#read] file the dervivative filestream optionally responds to :mime_type
-    # @option opts [RDF::URI or String] type URI for the RDF.type that identifies the file's role within the generic_file
+    # @param [#read or String] file the derivative filestream optionally responds to :mime_type
+    # @param [String] extract file type symbol (e.g. :thumbnail) from Hydra::Derivatives created destination_name
     # @option opts [Boolean] update_existing when set to false, always adds a new file.  When set to true, performs a create_or_update
     # @option opts [Boolean] versioning whether to create new version entries
 
-    def self.call(object, file, opts={})
-      type = opts.fetch(:type, ::RDF::URI("http://pcdm.org/use#ServiceFile"))
+    def self.call(object, file, destination_name, opts={})
```

Comment:
I don't believe opts can be filled in from the output_file_service.call from Hydra::Derivatives.  Only a custom processor would be able to fill this in now.  As it stood, the only opt that was ever passed was `:mime_type`, and that has been addressed.

Basically, `update_existing` and `versioning` always get set to `true` for persisting derivatives.  I would like to have `versioning` set to false instead.  I would not like to have versions of thumbnails.